OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -122,7 +122,12 @@ void INotifyAttack.PreparingAttack(Actor self, Target target, Armament a, Barrel
 
 		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel) { }
 
-		public virtual void Tick(Actor self)
+		void ITick.Tick(Actor self)
+		{
+			Tick(self);
```

Comment:
This seems to add more than the explicit interface?

Reply:
This is the only way to make ITick explicit without sacrificing the overridability.

Reply:
Note that we already use the same approach in various places (though more often for `INotifyCreated.Created`), so this is both consistent with other traits and proven & accepted as solutions for these cases.